src/Concordium/Client/Commands.hs

```diff
@@ -1348,10 +1410,71 @@ bakerAddCmd =
       )
       (progDesc "Deploy baker credentials to the chain."))
 
+allowedValuesOpenDelegationForAsString :: String
+allowedValuesOpenDelegationForAsString =
+    "'all' (delegators are allowed to join the pool), 'existing' (keep the existing delegators, but do not allow new delegators), 'none' (move existing delegators to passive delegation and do not allow new delegators)"
+
+openStatusFromStringInform :: String -> Either String OpenStatus
+openStatusFromStringInform "all" = Right OpenForAll
+openStatusFromStringInform "existing" = Right ClosedForNew
+openStatusFromStringInform "none" = Right ClosedForAll
+openStatusFromStringInform s = Left $
+    "Unexpected SELECTION '" ++ s ++ "'. The allowed values are: " ++ allowedValuesOpenDelegationForAsString
+
+helpOpenDelegationFor :: String
+helpOpenDelegationFor =
+    "Select whether the baker will allow other parties (delegators) to delegate CCD to the pool. Available values for SELECTION are: " ++ allowedValuesOpenDelegationForAsString ++ "."
+
+bakerConfigureCmd :: Mod CommandFields BakerCmd
+bakerConfigureCmd =
+  command
+    "configure"
+    (info
+      (BakerConfigure <$>
+        transactionOptsParser <*>
+        optional (option (eitherReader amountFromStringInform) (long "stake" <> metavar "CCD-AMOUNT" <> help "The amount of CCD to stake.")) <*>
+        optional (not <$> switch (long "no-restake" <> help "The earnings will not be added to the baker stake automatically.")) <*>
+        optional (option (eitherReader openStatusFromStringInform) (long "open-delegation-for" <> metavar "SELECTION" <> help helpOpenDelegationFor)) <*>
+        optional (strOption (long "baker-url" <> metavar "URL" <> help "A link to information about the baker.")) <*>
+        optional (option (eitherReader amountFractionFromStringInform) (long "delegation-transaction-fee-commission" <> metavar "DECIMAL-FRACTION" <> help ("Fraction the baker takes in commission from delegators on transaction fee rewards. " ++ rangesHelpString "transaction fee commission"))) <*>
+        optional (option (eitherReader amountFractionFromStringInform) (long "delegation-baking-commission" <> metavar "DECIMAL-FRACTION" <> help ("Fraction the baker takes in commission from delegators on baking rewards. " ++ rangesHelpString "baking reward commission"))) <*>
+        optional (option (eitherReader amountFractionFromStringInform) (long "delegation-finalization-commission" <> metavar "DECIMAL-FRACTION" <> help ("Fraction the baker takes in commission from delegators on finalization rewards. " ++ rangesHelpString "finalization reward commission"))) <*>
+        optional (
+            (,) <$>
+                strOption (long "keys-in" <> metavar "FILE" <> help "File containing baker credentials.") <*>
+                strOption (long "keys-out" <> metavar "FILE" <> help "File to write updated baker credentials to, in case of successful transaction. These can be used to start the node."))
```

Comment:
Is there a concrete reason why they are both required or none? This seems different from the `baker add` command which does not require keys-out even if keys-in is present.